core/src/main/java/com/predic8/membrane/core/kubernetes/KubernetesWatcher.java

```diff
@@ -87,8 +87,8 @@ private KubernetesClient getClient() {
 
     private Optional<KubernetesValidationInterceptor> findK8sValidatingInterceptor() {
         return router.getRules().stream()
-                .filter(rule -> rule.getInterceptors() != null)
                 .map(rule -> rule.getInterceptors())
+                .filter(interceptors -> interceptors != null)
```

Comment:
One line might be a duplicate:

.filter(interceptors -> interceptors != null)
.filter( i -> i != null)